src/CharTable.jl

```diff
@@ -130,14 +130,13 @@ struct SimpleCharTable{T} <: Table
 	ring::PolyRing  # Ring of polynomials of type T used in table
 	information::String  # General info about the table
 	chars::Vector{<:AbstractGenericCharacter}
-	irrchartypes::Int64  # Number of irreducible character types
```

Comment:
I'm a bit confused, I've already deleted this. Maybe this is based on an old branch?

Reply:
This is definitely present in `master`. Perhaps you only deleted this in the full char table type, not in the simple one?

Reply:
Oh, yes sorry.